### PR TITLE
Dashboard disease burden

### PR DESCRIPTION
**Pull request by danielrus**

### CODE REVIEW

File: src/app/shared/card/card-disease/card-disease.component.html [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+  <ul class="statistics">
+    <li class="statistics-cases">
+      <div [ngClass]="['count', progress ? 'positive ' + progress : '']">
+        <span class="current">{{ data.caseCount }}</span>
+        <span *ngIf="data.previousCaseCount" class="previous">
+          <mat-icon class="material-icons-round">{{ icon }}</mat-icon>
+          {{ data.previousCaseCount }}
+        </span>
+      </div>
+      <div class="label">Cases</div>
```
Comment (adriandumavitagroup) [inline src/app/shared/card/card-disease/card-disease.component.html]: translate

File: src/app/shared/card/card-disease/card-disease.component.html [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+    </li>
+    <li class="statistics-events">
+      <div class="count">
+        <span class="current">{{ data.eventCount }}</span>
+      </div>
+      <div class="label">Events</div>
```
Comment (adriandumavitagroup) [inline src/app/shared/card/card-disease/card-disease.component.html]: translate

File: src/app/shared/card/card-disease/card-disease.component.html [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+    </li>
+    <li class="statistics-fatalities">
+      <div class="count">
+        <span [ngClass]="{ current: true, positive: !!data.caseDeathCount }">
+          {{ data.caseDeathCount }}
+        </span>
+      </div>
+      <div class="label">Fatalities</div>
```
Comment (adriandumavitagroup) [inline src/app/shared/card/card-disease/card-disease.component.html]: translate

File: src/app/shared/card/card-disease/card-disease.component.html [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+    </li>
+  </ul>
+  <p *ngIf="data.lastReportedDistrictName" class="district">
+    <span class="label">Last report:</span>
```
Comment (adriandumavitagroup) [inline src/app/shared/card/card-disease/card-disease.component.html]: translate

**Review by sandorbanvitagroup — DISMISSED** [review]

**Review by sandorbanvitagroup — APPROVED** [review]

**Review by adriandumavitagroup — APPROVED** [review]

**Pull request merged**